templates/ccm/nutanix-ccm.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-ca-bundle
+  namespace: kube-system
+data:
+  ca.crt: NUTANIX_ADDITIONAL_TRUST_BUNDLE
```

Comment:
Did this pass e2e? Shouldn't this be a variable instead?

Reply:
See the `cluster-templates` makefile target; there's a `sed` command that replaces that.